### ISSUE TITLE
LUA error help

### ISSUE THREAD

**Issue by kitstsu** [jordonwow/bigdebuffs#214]
![Screenshot_28](https://user-images.githubusercontent.com/78232339/106319727-ef993d80-6282-11eb-802a-d47868d9f1b2.png)
it activates at different times. activated at different times. I have already reinstalled and cleaned the addon settings in WTF folder.

**Comment by flickamad** [comment]
I'm getting spammed by this error since yesterday and I don't know why, changed literally nothing.

**Comment by dinkletron** [comment]
Also getting this error. Usually when I pop alter time on my mage but has happened in various other scenarios as well since yesterday.

**Comment by tgray6** [comment]
Yeah, seems to be the Defensive Spells area I think, I removed the alter time from the Retail.lua file for now and seems to not trigger while playing anymore. Might be able to get it to work by just using different spellIds here, but not entirely sure:

    [342246] = { type = BUFF_DEFENSIVE }, --Alter Time (Arcane)
    [110909] = { type = BUFF_DEFENSIVE, parent = 108978 }, --Alter Time (Fire/Frost)

**Comment by Burncraze** [comment]
you have to change 
[342246] = { type = BUFF_DEFENSIVE }, --Alter Time (Arcane)
with
[342245] = { type = BUFF_DEFENSIVE }, --Alter Time (Arcane)

**Comment by domzae** [comment]
fixed in #206 
in the meantime you can go to line 240 in Addons/BigDebuffs/Retail.lua and change the parentID:

OLD
```
    [342246] = { type = BUFF_DEFENSIVE }, --Alter Time (Arcane)
        [110909] = { type = BUFF_DEFENSIVE, parent = 108978 }, --Alter Time (Fire/Frost) 
```
NEW
```
    [342246] = { type = BUFF_DEFENSIVE }, --Alter Time (Arcane)
        [110909] = { type = BUFF_DEFENSIVE, parent = 342246 }, --Alter Time (Fire/Frost)
```
